polaris-react/UNRELEASED.md

```diff
@@ -16,6 +16,7 @@ Use [the changelog guidelines](/documentation/Versioning%20and%20changelog.md) t
 - Replaced hard coded `font-size` and `line-height` values with tokens ([5355](https://github.com/Shopify/polaris/pull/5355/))
 - Replaced hard coded spacing values with tokens ([5364](https://github.com/Shopify/polaris/pull/5364/))
 - Simplified usage of color tokens ([5360](https://github.com/Shopify/polaris/pull/5360/))
+- Added optional visual density updates and zebra striping to `DataTable` ([5365](https://github.com/Shopify/polaris/pull/5365/))
```

Comment:
I know there are a few rows above without the `#` but pretty much every link to a PR includes the `#` -- can we update this and the rows above to include `#`?

```markdown
- Added optional visual density updates and zebra striping to `DataTable` ([#5365](https://github.com/Shopify/polaris/pull/5365/))
```